python/pyspark/mllib/tests.py

```diff
@@ -863,6 +863,22 @@ def test_model_transform(self):
             eprod.transform(sparsevec), SparseVector(3, [0], [3]))
 
 
+class LinearDataGeneratorTests(MLlibTestCase):
+    def test_dim(self):
+        points = LinearDataGenerator.generateLinearInput(
+            0.0, [0.0, 0.0, 0.0], [0.0, 0.0, 0.0],
+            [0.33, 0.33, 0.33], 4, 0, 0.1)
+        self.assertEqual(len(points), 4)
+        for point in points:
+            self.assertEqual(len(point.features), 3)
+
+        rdd = LinearDataGenerator.generateLinearRDD(
+            sc, 6, 2, 0.1, 2, 0.0).collect()
```

Comment:
Sam here. Use named arguments. The variable name `rdd` is confusing because this is already collected to driver.